queryx_test.go

```diff
@@ -149,3 +151,25 @@ func TestQueryxBindMap(t *testing.T) {
 		}
 	})
 }
+
+func TestQyeryxAllWrapped(t *testing.T) {
+	var (
+		gocqlQueryPtr = reflect.TypeOf((*gocql.Query)(nil))
+		queryxPtr     = reflect.TypeOf((*Queryx)(nil))
+	)
+
+	for i := 0; i < gocqlQueryPtr.NumMethod(); i++ {
+		m, ok := queryxPtr.MethodByName(gocqlQueryPtr.Method(i).Name)
+		if !ok {
+			t.Fatalf("Queryx missing method %s", gocqlQueryPtr.Method(i).Name)
```

Comment:
Having t.Errorf() here would end up with list of all missing methods. Fatal breaks test execution after first message.